app/models/state_file_notification_email.rb

```diff
@@ -16,13 +16,22 @@ class StateFileNotificationEmail < ApplicationRecord
   validates_presence_of :to
   validates_presence_of :body
   validates_presence_of :subject
+  validate :not_unsubscribed
 
   after_create_commit :deliver
 
   private
 
+  def not_unsubscribed
+    matching_intakes = StateFileAzIntake.where(email_address: to, unsubscribed_from_email: true) +
+      StateFileNyIntake.where(email_address: to, unsubscribed_from_email: true)
+    if matching_intakes.present?
+      DatadogApi.increment("mailgun.state_file_notification_emails.not_sent_because_unsubscribed")
```

Comment:
thought this might be interesting to capture. afaict, we shouldn't see any sentry errors from this failure to validate but someone tell me if i'm wrong...